tests/helpers/test_cli_config.py

```diff
@@ -32,9 +32,9 @@ def test_cli_config_write():
             "description": "Invenio RDM My Site Instance",
             "author_name": "CERN",
             "author_email": "info@my-site.com",
-            "year": "2020",
+            "year": "2022",
             "database": "postgresql",
-            "elasticsearch": "7",
+            "search_platform": "opensearch",
```

Comment:
minor: Since we still allow ES7, should we add a new test instead of modifying the existing one?